app/workers/vbms/submit_dependents_pdf_job.rb

```diff
@@ -14,9 +14,16 @@ def perform(saved_claim_id, va_file_number_with_payload, submittable_686, submit
       raise Invalid686cClaim unless claim.valid?(:run_686_form_jobs)
 
       claim.persistent_attachments.each do |attachment|
-        file_name = File.basename(attachment.file.url)
-        file_path = Common::FileHelpers.generate_temp_file(attachment.file.read, file_name)
+        file_extension = File.extname(URI.parse(attachment.file.url).path)
+        file_path = Common::FileHelpers.generate_temp_file(attachment.file.read)
+
+        if %w[.jpg .jpeg .png .pdf].include? file_extension.downcase
```

Comment:
> Are these the only types of files that should be allowed to be uploaded? Asking because it looks like it will only change the names on these files but still allow any file to be uploaded regardless of type. If that is intended behavior, please disregard.

Yes, but good catch - thank you for confirming.  We do limit the types of files that can be uploaded through the UI on the front end.  I've attached a screenshot in case that helps:

![Screen Shot 2021-05-26 at 9 29 49 AM](https://user-images.githubusercontent.com/53828142/119697593-31875a00-be05-11eb-9d1f-22d1a5bec1f8.png)